### DISCUSSION TITLE
Add Version Control Support

### DISCUSSION THREAD

**Discussion by FarshidShekari** [dbeaver/dbeaver#10720] [Ideas]
Hi
I think the title tells everything. I wanna able to connect an svn or a git repository and every time when I want to save my procedure this client do commits to the repository and then save changes into DBMS and every time I open this procedure it loads and displays a shortcut to changes histories.

**Comment by serge-rider** [comment]
It is already there.
You can install Git support using this instruction: https://github.com/dbeaver/dbeaver/wiki/Optional-extensions

Git repository: https://marketplace.eclipse.org/content/dbeaver-git

Note: EE version has this extension out of the box.
